### PR TITLE
Adds more eating noises

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
soundadd: Added several horrible eating noises for your pleasure.
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -0,0 +1,67 @@
+/obj/item/reagent_containers/food/snacks/attack(mob/M, mob/user, def_zone)
```
Comment (PopNotes) [inline hippiestation/code/modules/food_and_drinks/food/snacks.dm]: yes mirroring this whole thing is messy but it works god damn it

**Comment by 2Dollarbill** [comment]
Maybe. +1

File: hippiestation/code/modules/food_and_drinks/food/snacks.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/food_and_drinks/food/snacks.dm:1]: you are literally adding one line for a sound, just modify the original proc in the /tg/ code and add `// hippie - <thing you did>`

**Comment by dreamcrab** [comment]
why do i want horrible things in my game

i hate eating noises like smacking lips and chewing dont do this

**Comment by banthebantz** [comment]
smacks lips +1

File: hippiestation/code/modules/food_and_drinks/food/snacks.dm (hunk 1 above)
Comment (KayeArray) [reply hippiestation/code/modules/food_and_drinks/food/snacks.dm:1]: yes do what chnkr said

**Comment by PopNotes** [comment]
so you be sayin

**Comment by PopNotes** [comment]
build this

**Comment by McBawbaggings** [comment]
Too realistic sounding for spess imo

**Comment by PopNotes** [comment]
that's the idea boi

**Comment by GuyonBroadway** [comment]
remember once you apply byond sound variance it will sound cartooney.

**Comment by zamolxius** [comment]
really good,  make more sounds,  it makes space feel alive. Glub glub for carps when?  Barks when ian gets pet?

**Comment by Chnkr** [comment]
Your PR needs to be updated to reflect the current state of the master branch, please update your PR and either (re)open it, or ask a maintainer to (re)open it.

**Pull request closed without merging**